plugins/woocommerce/src/Blocks/BlockTypesController.php

```diff
@@ -464,6 +464,9 @@ protected function get_block_types() {
 			$block_types[] = 'ProductFilterClearButton';
 			$block_types[] = 'ProductFilterCheckboxList';
 			$block_types[] = 'ProductFilterChips';
+			if ( Features::is_enabled( 'isBlockifiedAddToCart' ) ) {
```

Comment:
My bad, the feature flag key is actually `blockified-add-to-cart`:
```suggestion
			if ( Features::is_enabled( 'blockified-add-to-cart' ) ) {
```